File: programme/negative-of-a-number/NegativeOfANumber.java
// Write a programme to find negative of a number

import java.util.Scanner;

public class NegativeOfANumber {

  public static double getOpposite(double number) {
    return number *= -1;
  }

  public static void main(String[] args) {
    var sc = new Scanner(System.in);
    System.out.print("Input  : ");
    double number = sc.nextDouble();
    sc.close();
    System.out.println("Output : " + getOpposite(number));
  }
}

File: programme/negative-of-a-number/negative-of-a-number.go
// Write a programme to find negative of a number
package main

import (
	"fmt"
	"strconv"
)

func main() {
	var str string

	fmt.Print("Input Number : ")
	fmt.Scan(&str)

	num, err := strconv.Atoi(str)
	if err != nil {
		fmt.Printf("Input error: %s", str)
		return
	}

	if num > 0 {
		num = -num
	}

	fmt.Printf("\nOutput: %d\n", num)
}
